tools/checked-c-convert/ConstraintBuilder.cpp

```diff
@@ -311,11 +317,62 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
           constrainEq(ParameterEC, ParameterDC, Info);
         } else {
           // Constrain ParameterEC to wild if it is a pointer type.
-          llvm_unreachable("TODO");
+          Constraints &CS = Info.getConstraints();
+          for (const auto &C : ParameterEC)
+            C->constrainTo(CS, CS.getWild());
         }
 
         i++;
       }
+    } else if (DeclaratorDecl *DD = dyn_cast<DeclaratorDecl>(D)){
+      // This could be a function pointer.
```

Comment:
This comment is a little confusing: it could be a function pointer or a function, right?